src/order/atoms.lean

```diff
@@ -683,15 +683,8 @@ begin
   suffices : (∀ b : α, b = ⊥ ∨ ∃ (a : α), is_atom a ∧ a ≤ b) ↔
     (∀ b : β, b = ⊥ ∨ ∃ (a : β), is_atom a ∧ a ≤ b),
   from ⟨λ ⟨p⟩, ⟨this.mp p⟩, λ ⟨p⟩, ⟨this.mpr p⟩⟩,
```

Comment:
This part could also be golfed if you add the `@[mk_iff]` attribute to `is_atomic`, so that it generates a `is_atomic_iff` lemma for you. (Same for `is_coatomic`.)

Reply:
Done.